dotcom-rendering/src/palette.ts

```diff
@@ -13,6 +13,36 @@ import { decidePalette } from './lib/decidePalette';
 import { transparentColour } from './lib/transparentColour';
 
 // ----- Palette Functions ----- //
+/**
+ * Picks a variant of colour for a palette corresponding to the theme
+ * @param theme
+ * @param variant
+ */
+const pillarPalette = (
+	theme: ArticleTheme,
+	variant: 300 | 400 | 500,
+): string => {
+	switch (theme) {
+		case Pillar.News:
+			return sourcePalette.news[variant];
+		case Pillar.Lifestyle:
+			return sourcePalette.lifestyle[variant];
+		case Pillar.Sport:
+			return sourcePalette.sport[variant];
+		case Pillar.Culture:
+			return sourcePalette.culture[variant];
+		case Pillar.Opinion:
+			return sourcePalette.opinion[variant];
+		case ArticleSpecial.Labs:
+			return sourcePalette.specialReport[variant];
+		case ArticleSpecial.SpecialReport:
+			return sourcePalette.specialReport[variant];
+		case ArticleSpecial.SpecialReportAlt:
+			return sourcePalette.news[variant];
+		default:
+			return sourcePalette.news[variant];
+	}
+};
```

Comment:
If TypeScript is happy that `300 | 400 | 500` exist, I am!

The word that describes how light or dark a colour is is actually [“lightness” or “value”](https://en.wikipedia.org/wiki/Lightness) rather than variant.